scripts/safe_template_linter.py

```diff
@@ -836,13 +842,21 @@ def _check_html_and_text(self, expression, results):
                         if html_inner_start_index <= string.start_index and string.end_index <= html_inner_end_index:
                             unwrapped_html_strings.remove(string)
 
-        # check strings not wrapped in HTML()
+        # check strings not wrapped in HTML() for '<'
         for string in unwrapped_html_strings:
             if '<' in string.string_inner:
                 results.violations.append(ExpressionRuleViolation(
                     Rules.mako_wrap_html, expression
                 ))
                 break
+        # check strings not wrapped in HTML() for HTML entities
+        if has_page_default:
+            for string in unwrapped_html_strings:
+                if re.search(r"&[^\s]+;", string.string_inner):
```

Comment:
They will. But, I could limit the valid characters more I guess.
http://www.w3schools.com/html/html_entities.asp